iio-widgets/src/datastrategy/filedemodatastrategy.cpp

```diff
@@ -0,0 +1,27 @@
+#include "datastrategy/filedemodatastrategy.h"
+#include <utility>
+#include <QFile>
+
+using namespace scopy::attr;
+
+FileDemoDataStrategy::FileDemoDataStrategy(IIOWidgetFactoryRecipe recipe, QObject *parent)
+{
+	m_recipe = std::move(recipe);
```

Comment:
std::move caused issues previously. Are you sure this is needed here ? Make sure you update this to all necessary classes.